src/components/BookSocialTextField.js

```diff
@@ -1,52 +1,66 @@
 import React from 'react';
-import {TextField, Typography} from '@mui/material'
+import {TextField} from '@mui/material';
+import PropTypes from 'prop-types';
 import paletteColors from "../resources/palette";
-import PropTypes from 'prop-types'
-
 
 const BookSocialTextField = ({
-    label,
-    value,
-    onChange,
-    errorMessage = "",
-    status = "default", // 'default', 'error', or 'success'
-}) => {
+                                 value,
+                                 onChange,
+                                 label,
+                                 type,
+                                 sx = {},
```

Comment:
I'd say it's a better practice not to add the sx and put everything that can be modifiable with default values in attributes. This way we protect the widget from being inconsistent, in case you want to add padding or so on you can do it in the context of the page, what do you think?